Binary_Insertion_Sort/Binary_Insertion_Sort.cs

```diff
@@ -0,0 +1,51 @@
+/*We can use binary search to reduce the number of comparisons in normal insertion sort. Binary Insertion Sort uses binary search to find the proper location to insert the selected item at each iteration.*/
```

Comment:
Please split the comment to span multiple lines as it is going out of screen currently. Try not to exceed 80 characters per line.